src/api/db/data_schema.rb

```diff
@@ -1 +1 @@
-DataMigrate::Data.define(version: 20250312082028)
+DataMigrate::Data.define(version: 20250326100650)
```

Comment:
I miss a file with the data migration file...

Reply:
ups, pushed

Reply:
I forgot to mention. We also would need to include a couple of lines in `src/api/db/seeds.rb`, for new instances, if I'm not mistaken, like in f7a3795d619f8b2c166a03359bb12106ef6c1930.

Reply:
right, added as well now.